src/backends/backend/triton_model.cc

```diff
@@ -171,9 +197,17 @@ TritonModel::Create(
 
   // Create and initialize the model.
   std::unique_ptr<TritonModel> local_model(new TritonModel(
-      server, localized_model_dir, backend, min_compute_capability));
-  RETURN_IF_ERROR(
-      local_model->Init(version_path, model_config, "" /* platform */));
+      server, localized_model_dir, backend, min_compute_capability,
+      auto_complete_config));
+  // If request for auto completion, Init() will be postponed until
+  // UpdateModelConfig() is called as Init() assumes the model config
+  // is well-formed.
```

Comment:
This isn't ideal because if the backend never calls SetModelConfig then Init will not be called. I think it it ok for now as this complicated flow will go away once all backends are moved over and we can get rid of the InferenceBackend class.

Reply:
Okay. I added a FIXME comment as reminder.